libmscore/layout.cpp

```diff
@@ -3767,9 +3768,12 @@ void Score::layoutSystemElements(System* system, LayoutContext& lc)
                                     QRectF r = f->bbox().translated(f->pos() + n->pos() + n->chord()->pos() + s->pos() + s->measure()->pos());
                                     system->staff(f->note()->chord()->vStaffIdx())->skyline().add(r);
                                     }
+                                    recreateShapes.insert(f->staffIdx());
```

Comment:
indent is off, though... actually I guess it's arguable as to whether this belongs within the "if" clause, maybe the regeneration isn't needed otherwise if we aren't adding the fingering to the skyline?  but probably better safe than sorry

Reply:
I was puzzled whether this statement should be inside or outside the loop. Some test didn't show any differences after playing with <code>Autoplace</code> and <code>Visible</code> (that's what <code>addToSkyline()</code> is testing for.
But indeed, better save then sorry, I decided to move <code>insert</code> outside the <code>if</code> . But unfortunately I forgot the indent (can't get used to this style :-)).